### PR TITLE
Fixing balance card UI

### PR DESCRIPTION
**Pull request by zaelgohary**
Related issue: https://github.com/threefoldtech/tfgrid_dashboard/issues/239

### CODE REVIEW

File: src/portal/components/FundsCard.vue [hunk 1]
```diff
@@ -135,12 +135,12 @@ export default class FundsCard extends Vue {
 }
 </script>
 <style scoped>
-.fund {
+/* .fund {
```
Comment (rawdaGastan) [inline src/portal/components/FundsCard.vue]: We can remove it.

File: src/portal/components/FundsCard.vue [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <template>
   <v-container>
-    <v-card class="fund d-flex align-center font-weight-bold mr-4 primary" style="margin-right: 10% !important;">
+    <v-card class="fund d-flex align-center font-weight-bold mr-4 primary">
```
Comment (rawdaGastan) [inline src/portal/components/FundsCard.vue]: Also here we can remove `fund` class

File: src/portal/Portal.vue [hunk 3]
```diff
@@ -21,29 +15,7 @@ import { Component, Vue, Watch } from "vue-property-decorator";
   components: { FundsCard },
```
Comment (rawdaGastan) [inline src/portal/Portal.vue]: Can we remove the unused component `FundsCard`?
and the unused imports

File: src/portal/Portal.vue [hunk 4]
```diff
@@ -21,29 +15,7 @@ import { Component, Vue, Watch } from "vue-property-decorator";
...
-  updated() {
-    this.balanceFree = this.$route.query.balanceFree;
-    this.balanceReserved = this.$route.query.balanceReserved;
-  }
-  unmounted() {
-    this.$store.dispatch("portal/unsubscribeAccounts");
-  }
 }
 </script>
 <style scoped>
```
Comment (rawdaGastan) [inline src/portal/Portal.vue]: Here I think we don't need the style

File: src/Dashboard.vue [hunk 5]
```diff
@@ -307,7 +322,37 @@ export default class Dashboard extends Vue {
...
+
+  @Watch("this.$route.query.balance") async onBalanceUpdate(
+    value: number,
+    oldValue: number
+  ) {
+    this.balanceFree = this.$route.query.balanceFree;
+    this.balanceReserved = this.$route.query.balanceReserved;
+    console.log(`balance went from ${oldValue}, to ${value}`);
+  }
+  // mounted() {
```
Comment (rawdaGastan) [inline src/Dashboard.vue]: Can we remove the commented code ?

File: src/Dashboard.vue [hunk 6]
```diff
@@ -10,77 +10,90 @@
       >
         <v-app-bar-nav-icon></v-app-bar-nav-icon>
 
-        <v-toolbar-title
+        <!-- <v-toolbar-title
```
Comment (rawdaGastan) [inline src/Dashboard.vue]: why you commented this part?

**Review by rawdaGastan — CHANGES_REQUESTED** [review]

File: src/Dashboard.vue (hunk 6 above)
Comment (zaelgohary) [reply src/Dashboard.vue]: I thought it was duplicated with the title in the side menu. I uncommented it.

File: src/Dashboard.vue (hunk 5 above)
Comment (zaelgohary) [reply src/Dashboard.vue]: done.

File: src/portal/Portal.vue (hunk 4 above)
Comment (zaelgohary) [reply src/portal/Portal.vue]: done.

File: src/portal/components/FundsCard.vue (hunk 2 above)
Comment (zaelgohary) [reply src/portal/components/FundsCard.vue]: done.

File: src/portal/Portal.vue (hunk 3 above)
Comment (zaelgohary) [reply src/portal/Portal.vue]: done.

File: src/portal/components/FundsCard.vue (hunk 1 above)
Comment (zaelgohary) [reply src/portal/components/FundsCard.vue]: yup!

File: src/portal/Portal.vue (hunk 3 above)
Comment (rawdaGastan) [reply src/portal/Portal.vue]: can we remove this line? and its import

Comment (zaelgohary) [reply src/portal/Portal.vue]: done.

**Review by rawdaGastan — APPROVED** [review]

**Pull request merged**